README.md

```diff
@@ -15,19 +15,19 @@ ODBC or ADO.NET, which are mostly old and static standards and create
 additional complexity due to the necessary driver managers.
 
 ## Changes
+* [WebSocket API v4](docs/WebsocketAPIV4.md) support has been added in Exasol 8.31.0.
 * Autocommit was enabled by default for sessions in Exasol 7.1+.
 * [WebSocket API v3](docs/WebsocketAPIV3.md) support has been added in Exasol 7.1.
-* [WebSocket API v2](docs/WebsocketAPIV2.md) support has been added in Exasol 7.0.
```

Comment:
Is it intended to be removed?

Reply:
I removed it because 7.0 isn't supported anymore.

Reply:
ok.